### ISSUE TITLE
Legeacy Blocks.Json missing from release?

### ISSUE THREAD

**Issue by Morketh** [matcool/anvil-parser#17]
in legacy.py theres a call to open a file ``open(os.path.join(os.path.dirname(__file__), 'legacy_blocks.json')`` however manually inspecting the site packages folder for Visual Studios that file does not exist.
![image](https://user-images.githubusercontent.com/6291631/91498796-a2e9aa80-e87d-11ea-996c-4a74751f816b.png)
I was able to solve this by manually pulling the file from github as its been included in the repository. 
Windows:
```
  Message=[Errno 2] No such file or directory: 'C:\\Program Files (x86)\\Microsoft Visual Studio\\Shared\\Python37_64\\lib\\site-packages\\anvil\\legacy_blocks.json'
  Source=E:\LinuxShare\repos\mc-pov\RenderFarm\RenderFarm.py
  StackTrace:
  File "E:\LinuxShare\repos\mc-pov\RenderFarm\RenderFarm.py", line 7, in <module>
    import anvil
```

Linux: Installed using pip
```
admiral@USS-Havok:/opt/progs/repos/mc-pov/RenderFarm$ python3 RenderFarm.py 
Traceback (most recent call last):
  File "RenderFarm.py", line 7, in <module>
    import anvil
  File "/home/admiral/.local/lib/python3.6/site-packages/anvil/__init__.py", line 1, in <module>
    from .chunk import Chunk
  File "/home/admiral/.local/lib/python3.6/site-packages/anvil/chunk.py", line 3, in <module>
    from .block import Block, OldBlock
  File "/home/admiral/.local/lib/python3.6/site-packages/anvil/block.py", line 3, in <module>
    from .legacy import LEGACY_ID_MAP
  File "/home/admiral/.local/lib/python3.6/site-packages/anvil/legacy.py", line 4, in <module>
    with open(os.path.join(os.path.dirname(__file__), 'legacy_blocks.json'), 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: '/home/admiral/.local/lib/python3.6/site-packages/anvil/legacy_blocks.json'```

**Comment by matcool** [comment]
my bad, i didn't know you had to manually include non `.py` files. fixed in 0.8.2

**Comment by Morketh** [comment]
I was also not aware that needed to be done. thanks for teaching me something new today.
